data/json/items/armor/torso_clothes.json

```diff
@@ -1670,7 +1670,7 @@
       },
       {
         "id": "swordocracy_tshirt",
-        "name": { "str": "Down with Swordocracy t-shirt", "str_pl": "Down with Swordocracy t-shirts" },
+        "name": { "str": "Down with Swordocracy t-shirt" },
```

Comment:
The plural is here to suppress the warning about potential irregular plural form because the name contains `with`.

Reply:
So these should remain as is?

Reply:
Yes.